### DISCUSSION TITLE
nova eloquent table-less data from external source

### DISCUSSION THREAD

**Discussion by xerox-xeon** [laravel/nova-issues#3315] [Q&A] [answered]
I want to display table via external data from HTTP rest API 

like this:https://stackoverflow.com/questions/47772103/laravel-eloquent-table-less-data-from-external-source

**Comment by crynobone** [comment, marked as answer]
Laravel Nova can only handle Eloquent based model. 

To do this you might need to use some tool to convert API response to Eloquent, maybe using https://github.com/calebporzio/sushi

Also you may look into creating custom Tool to handle displaying external API data.
